### DISCUSSION TITLE
Any props to define threshold when we will pre-fetch data?

### DISCUSSION THREAD

**Discussion by ilhamwahabi** [petyosi/react-virtuoso#578] [Q&A]
In `react-virtualized` we have ["threshold" props](https://github.com/bvaughn/react-virtualized/blob/master/docs/InfiniteLoader.md#prop-types) which is `Threshold at which to pre-fetch data. A threshold X means that data will start loading when a user scrolls within X rows. Defaults to 15.`

Do we have something similar in `react-virtuoso`?

**Comment by VladSez** [comment]
Yep, I am also interested in this prop in case of [endless scrolling](https://virtuoso.dev/endless-scrolling) scenario.
